### PR TITLE
Fix type of parser

### PR DESCRIPTION
**Pull request by gendosu**
Fix type of parser in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ in:
   type: file
   path_prefix: example.txt
   parser:
-    type: parser
+    type: none
     format: none
```
Comment (sonots) [inline README.md]: Ah, this line is also wrong. Could you fix together?

Comment (gendosu) [inline README.md]: [format: none] may be erased?

Comment (sonots) [inline README.md]: yes

Comment (gendosu) [inline README.md]: Roger

**Comment by sonots** [comment]
Thanks!

**Pull request merged**